NewsBook/Repository/FavouriteNewsRespository.cs

```diff
@@ -42,8 +42,9 @@ IIdentityServices identityServices
             );
         }
 
-        public async Task<FavouriteNews> Insert(Guid newsId, bool isFavourite = true)
+        public async Task<FavouriteNews> Insert(Guid newsId)
```

Comment:
why have we removed `isFavourite` parameter from the method? Is there any specific reason?